truss/tests/test_truss_handle.py

```diff
@@ -115,11 +115,8 @@ def _generate_base_image_variations(
             (base_template.format(version, version_tag), base_py3_path, False)
         )
 
-        # NB(nikhil): Python 3.8 base images currently don't have support for development models
-        # on GPUs.
-        fail = False if version != "3.8" else True
```

Comment:
This used to fail because of the way that we installed the `venv` for the control server, but it actually works with `uv` now. We actually weren't catching that this was _not_ failing until I changed the assertions below, and that's how I caught this